net.oz9aec.Gpredict.json

```diff
@@ -0,0 +1,63 @@
+{
+	"app-id": "net.oz9aec.Gpredict",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.28",
+	"sdk": "org.gnome.Sdk",
+	"command": "gpredict",
+	"finish-args": [
+		"--share=ipc", "--socket=x11",
+		"--socket=wayland",
+		"--share=network", "--persist=.config/Gpredict"
+	],
+	"build-options" : {
+		"cflags": "-O2 -g",
+		"cxxflags": "-O2 -g"
+	},
+	"cleanup": ["/include", "/lib/debug", "/share/man", "/share/gtk-doc", "/share/doc/hamlib" ],
+	"modules": [
+		{
+			"name": "libgoocanvas",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://ftp.gnome.org/pub/GNOME/sources/goocanvas/2.0/goocanvas-2.0.4.tar.xz",
+					"sha256": "c728e2b7d4425ae81b54e1e07a3d3c8a4bd6377a63cffa43006045bceaa92e90"
+				}
+			]
+		},
+		{
+			"name": "hamlib",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://sourceforge.net/projects/hamlib/files/hamlib/3.2/hamlib-3.2.tar.gz",
+					"sha256": "b55cb5e6a8e876cceb86590c594ea5a6eb5dff2e30fc13ce053b46baa6d7ad1d"
+				}
+			]
+		},
+		{
+			"name": "appdata",
```

Comment:
Should be contributed upstream (and before that, it can be added just as `file` resource under gpredict).